scripts/portfolioView.js

```diff
@@ -0,0 +1,50 @@
+(function(module) {
+
+  var projectView = {};
+
+  projectView.mainNav = function() {
+    $('.head-nav').on('click', 'li', function () {
+      $('.tab-content').hide();
+      // $('#' + $(this).data('content')).show();
+      $('#' + $(this).data('content')).fadeIn(); //I recommended that Max use a .fadeIn() method, to make the change between single tabs seem smoother.
+    });
+
+    $('.head-nav .tab:first').click();
+  };
+
+  projectView.populateFilters = function() {
+    $('article').each(function() {
+      if (!$(this).hasClass('template')) {
+        val = $(this).attr('data-category');
+        var optionTag = '<option value="' + val + '">' + val + '</option>';
+        if ($('#category-filter option[value="' + val + '"]').length === 0) {
+          $('#category-filter').append(optionTag);
+        }
+      }
+    });
+  };
+
+  projectView.categoryFilter = function() {
+    $('#category-filter').on('change', function() {
+      if ($(this).val()) {
+        $('article').hide();
+        $('article[data-category="' + $(this).val() + '"]').fadeIn();
+      } else {
+        $('article').fadeIn();
+        // $('project.template').hide();
+      }
+    });
+  };
+
+  projectView.initIndexPage = function() {
+    Project.all.forEach(function(z){
+      $('#projects').append(z.toHtml());
+    });
+
+    projectView.mainNav();
+    projectView.populateFilters();
+    projectView.categoryFilter();
+  };
+
+  module.projectView = projectView;
+})(window);
```

Comment:
KL: Overall, good use of concepts from class. Everything is structured where I could easily read and follow what you coded. Visually, I suggest you use a color scheme where the colors match a bit better. But other than that, everything looks good!